cluster/images/conformance/go-runner/run-conformance.go

```diff
@@ -0,0 +1,341 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
This file is borderline too big, and so you want to pigeon hole it to conformance or just make conformance be the default?

Reply:
I'm a little confused what the question is; I think maybe you're pointing out the fact that the file is named `run-conformance` and seeing if that is related to a focus on just the `Conformance` tests and not the e2e in general?

No, the name was just a result of it going into the `conformance` image even though it runs all the e2e tests.

I'm not sure what you're getting at about the file size; could you clarify?

Reply:
1 - This file should be broken into files.  
2 - change the name to something like e2erunner or something more generic.  The k8s code base generally does not use hyphenated names.